apps/frontend/src/pages/quiz/styles.module.css

```diff
@@ -17,18 +21,105 @@
 }
 
 .header {
-	max-width: 300px;
+	max-width: 458px;
+	font-size: 48px;
+	line-height: 64px;
 	text-align: center;
 }
 
 .text {
-	max-width: 470px;
-	font-size: 20px;
+	max-width: 722px;
+	font-size: 32px;
 	font-weight: 700;
+	line-height: 40px;
 	text-align: center;
 }
 
 .btn {
-	width: 100%;
-	max-width: 250px;
+	width: 322px;
+}
+
+.wheel-img {
+	width: 220px;
+	height: 205px;
+}
+
+.ripple-effect__background1,
+.ripple-effect__background2 {
+	display: none;
+}
+
+@media (width <= 640px) {
```

Comment:
Can we extract the `640px` value to some `breakpoints` constants? `640px` should be considered a `small` breakpoint. We don't have any other specific designs for other breakpoints, but there should be a possibility of adding one if needed.

Reply:
I figured out that CSS variables can only be used in properties and can't be used in media queries (it is possible with preprocessors, but we don't use them). I haven't found another way to extract the breakpoint value into a constant. Do you have any ideas on how this could be done?